_includes/comparison.html

```diff
@@ -214,6 +232,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td></td>
         <td></td>
         <td><a href="https://docs.microsoft.com/en-us/azure/aks/servicemesh-osm-about" target="_blank">Microsoft Azure</a></td>
+        <td></td>
```

Comment:
Cilium is directly integrated without requiring further installation into: EKS-A, GKE, Anthos, and DigitalOcean
All other cloud environments are supported via the default installation path

Reply:
This section is about integrations on the cloud provider sides that provide config checksm automatic updates, etc.